consul-secure-acl-vault/step3.md

```diff
@@ -0,0 +1,55 @@
+
+### Create Consul server policy
+
+Once the ACL system is initialized, if you check Consul logs again you should find some warnings in your logs:
+
+`cat ~/log/consul.log`{{execute T1}}
+
+```
+ [WARN]  agent: Node info update blocked by ACLs: node=c19d6af9-760b-a3cc-bbdd-b4f7209a79de accessorID=00000000-0000-0000-0000-000000000002
+ [WARN]  agent: Coordinate update blocked by ACLs: accessorID=00000000-0000-0000-0000-000000000002
+```
+
+These warnings are ACL logs from which you can infer the following: 
+- one node is trying to update its info in your Consul datacenter and gets denied access by the ACL system.
+- the node is not using any token to perform the request (because `00000000-0000-0000-0000-000000000002` is the anonymous policy used when no token is presented)
```

Comment:
```suggestion
- the agent does not have an assigned token to perform the request (because `00000000-0000-0000-0000-000000000002` is the anonymous policy used when no token is presented)
```